### DISCUSSION TITLE
Apple Watch Integration

### DISCUSSION THREAD

**Discussion by PasChris** [TortugaPower/BookPlayer#590] [Q&A] [answered]
I'm new to bookplayer, but haven't seen this question here. I'm running it on my iphone and apple watch. Is there a way, or a plan, to be able to sync a book with my watch so that I can leave my phone and still listen to the book?

**Comment by pichfl** [comment]
This question came up a few times, #464 was the last one. We are working on a solid sync model which will allow syncing to and playing from the Apple Watch directly :)

**Reply by PasChris** [reply]
Thanks! Looking forward to it. Loving the app so far

**Comment by tmills1073** [comment]
Hey, i LOVE this app.  Just wondering if this is still in the works.  I see it is still open in the, but curious if theres any movement. I'm not a dev, so just asking, and don't want to be impolite.  Just looking forward to using the app on the watch.  Thanks!

**Comment by lambdakilo** [comment]
Isn't this already a feature? I have atleast the BookPlayer application on my Apple Watch.

**Comment by GianniCarlo** [comment, marked as answer]
This is now live on v5.5.0 for existing subscribers 🎉

**Reply by veitch123** [reply]
I'm on 5.6 TestFlight but see no way to sync my place in the books. Am I doing something wrong ?

**Reply by GianniCarlo** [reply]
@veitch123 I think it's best if you switch back to the AppStore version, I'm not keeping the TestFlight betas up to date regularly, unless there's a bug I'm actively working on and getting quick feedback on a ticket.

That said, on your Apple Watch, you have to sign in, and with that it'll query our servers directly for your library info, and for any audio file you want to stream or download onto your Watch (swipe left on an item row, and download it) when you hit play on any of your books
